jobs/nginx/anime-manga-soc.hcl

```diff
@@ -29,17 +29,22 @@ job "nginx-ams" {
         "traefik.enable=true",
         "traefik.http.routers.nginx-ams.rule=Host(`amikon.me`,`www.amikon.me`)",
         "traefik.http.routers.nginx-ams.entrypoints=web,websecure",
-        "traefik.http.routers.nginx-ams.tls.certresolver=lets-encrypt"
+        "traefik.http.routers.nginx-ams.tls.certresolver=lets-encrypt",
       ]
     }
 
     task "webserver" {
       driver = "docker"
 
       config {
-        image = "ghcr.io/dcuams/amikon-website"
+        image = "ghcr.io/dcuams/amikon-site-v2:latest"
         ports = ["http", "https"]
```

Comment:
And only "http" is needed here, but still as an array

Reply:
made those changes, just going to test them

Reply:
I believe the brackets still need to be there, [see here](https://github.com/DistroByte/nomad/blob/8996eb4c0317001ae8d684dcdbb7424c80cb7abf/jobs/web/website.hcl#L36)